modules/currency-l1/src/main/scala/org/tessellation/currency/l1/cli/method.scala

```diff
@@ -67,6 +67,7 @@ object method {
 
     val l0SeedlistPath = None
 
+    val prioritySeedlistPath = None
```

Comment:
Do we need this to have a default value (None) given that we are strictly controlling when and to what value this variable is set?

Reply:
changed